conda_smithy/lint_recipe.py

```diff
@@ -122,6 +122,20 @@ def lintify(meta, recipe_dir=None):
         lints.append('The recipe `license` should not include the word '
                      '"License".')
 
+    # 11: There should be one empty line at the end of the file.
+    if recipe_dir is not None and os.path.exists(meta_fname):
+        with open(meta_fname, 'r') as f:
+            lines = f.read().split('\n')
+
+        end_empty_lines_count = 0
+        for i, line in enumerate(reversed(lines)):
+            if line != '':
+                break
+            end_empty_lines_count = i + 1
+
+        if lines[-1] != '' or end_empty_lines_count != 1:
```

Comment:
No, check 1 takes into account that there is no empty line at the end

Check 2 takes into account more than 1 empty line at the end